### PR TITLE
Rails setup

### PR DESCRIPTION
**Pull request by Hisoka37**
## Hello CR 😄 

In this PR I:

- Create a new rails app.
- Initialize your project with Git.
- Make sure that project has a Postgres database set up.
- Build project schema.
    - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
    - Include Foreign keys

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.integer 'user_id'
+    t.integer 'post_id'
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The comments table should have an `author_id` column, not `user_id`. Kindly rename this column.

- [ ] The comments table should also have a corresponding index for the `author_id`. Kindly add that too.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  create_table 'likes', force: :cascade do |t|
+    t.integer 'user_id'
+    t.integer 'post_id'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The likes table should have an `author_id` column, not `user_id`. Kindly rename this column.

- [ ] The likes table should also have a corresponding index for the `author_id`. Kindly add that too.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.text 'text'
+    t.integer 'comments_counter'
+    t.integer 'likes_counter'
+    t.integer 'author_id'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The posts table should also have a corresponding index for the `author_id`. Kindly add that too.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Hisoka37,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @Hisoka37 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Data table and references are correctly done

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**